<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-RT8ROG2F" failures="0" tests="2" name="Default test" time="0.158" errors="0" timestamp="27 Apr 2023 15:27:00 GMT">
  <testcase classname="testcases.BaseClass1" name="@BeforeTest ReportSetup" time="0.13">
    <failure type="com.codoid.products.exception.FilloException" message="Workbook is not found - SwagLabsmaven.xlsx">
      <![CDATA[com.codoid.products.exception.FilloException: Workbook is not found - SwagLabsmaven.xlsx
at com.codoid.products.fillo.Fillo.getConnection(Fillo.java:46)
at testcases.BaseClass1.ReportSetup(BaseClass1.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest ReportSetup -->
  <testcase classname="testcases.BaseClass1" name="@BeforeMethod SetUp" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod SetUp -->
  <testcase classname="testcases.BaseClass1" name="@AfterMethod TearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase classname="testcases.BaseClass1" name="@BeforeMethod SetUp" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod SetUp -->
  <testcase classname="testcases.BaseClass1" name="@AfterMethod TearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase classname="testcases.BaseClass1" name="@AfterTest ReportClean" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest ReportClean -->
  <testcase classname="testcases.Logintest" name="InvalidLogin" time="0.0">
    <skipped/>
  </testcase> <!-- InvalidLogin -->
  <testcase classname="testcases.Logintest" name="ValidLogin" time="0.0">
    <skipped/>
  </testcase> <!-- ValidLogin -->
  <testcase classname="testcases.BaseClass1" name="@BeforeTest ReportSetup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest ReportSetup -->
  <testcase classname="testcases.BaseClass1" name="@AfterTest ReportClean" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest ReportClean -->
</testsuite> <!-- Default test -->
